general/g.findfile/main.c

```diff
@@ -74,6 +75,12 @@ int main(int argc, char *argv[])
     l_flag->description = _("List available elements and exit");
     l_flag->suppress_required = YES;
 
+    t_flag = G_define_flag();
+    t_flag->key = 't';
+    t_flag->label = _("Return code 0 when file found, 1 otherwise");
```

Comment:
What is the terminology, exit code or return code? Is it clear for the translators?

Reply:
Exit status and return code are the same thing, I think it would be difficult to guess what translators are more familiar with. I copied this from r.mask.status.

Reply:
Fine like this